Wrappers/Python/cil/io/RAWFileWriter.py

```diff
@@ -0,0 +1,191 @@
+        Parameters
+        ----------
+        data : DataContainer, AcquisitionData or ImageData
+            This represents the data to save to TIFF file(s)
+        file_name : string
+            This defines the file name prefix, i.e. the file name without the extension.
+        compression : str, default None. Accepted values None, 'uint8', 'uint16'
+            The lossy compression to apply. The default None will not compress data. 
+            'uint8' or 'unit16' will compress to unsigned int 8 and 16 bit respectively.
+
+
+
+        Example of using the writer with compression to `uint8`:
+
+        Example
+        -------
+        
+        >>> from cil.io import RAWFileWriter
+        >>> writer = RAWFileWriter(data=data, file_name=fname, compression='uint8')
+        >>> writer.write()
+
+        Example
+        -------
+
+        Example of reading the data from the ini file:
+        
+        >>> config = configparser.ConfigParser()
+        >>> inifname = "file_name.ini"
+        >>> config.read(inifname)
+        >>> read_dtype = config['MINIMAL INFO']['data_type']
+        >>> read_array = np.fromfile(fname, dtype=read_dtype)
+        >>> read_shape = eval(config['MINIMAL INFO']['shape'])
+        >>> scale = float(config['COMPRESSION']['scale'])
+        >>> offset = float(config['COMPRESSION']['offset'])
+
+        Note
+        ----
+
+          If compression ``uint8`` or ``unit16`` are used, the scale and offset used to compress the data are saved 
+          in a file called ``scaleoffset.json`` in the same directory as the TIFF file(s).
+
+          The original data can be obtained by: ``original_data = (compressed_data - offset) / scale``
+
+        Note
+        ----
+
+          Data is always written in ‘C’ order independent of the order of the original data, 
+          https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tofile.html#numpy.ndarray.tofile, 
+          
+                
+    '''
+    
+    def __init__(self, data, file_name, compression=None):
+        
+        if not isinstance(data, (ImageData, AcquisitionData) ):
+            raise Exception('Writer supports only following data types:\n' +
+                            ' - ImageData\n - AcquisitionData')
+
+        self.data_container = data
+        file_name = os.path.abspath(file_name)
+        self.file_name = os.path.splitext(
+            os.path.basename( file_name )
+            )[0]
+        
+        self.dir_name = os.path.dirname(file_name)
+        logger.info ("dir_name {}".format(self.dir_name))
+        logger.info ("file_name {}".format(self.file_name))
+        
+        # Deal with compression
+        self.compress           = utilities.get_compress(compression)
+        self.dtype              = utilities.get_compressed_dtype(data, compression)
+        self.scale, self.offset = utilities.get_compression_scale_offset(data, compression)
+        self.compression        = compression
+    
+    def write(self):
```

Comment:
Maybe a docstring is needed on this method for it to be rendered in the documentation

Reply:
that's correct